en/sim_gazebo_gz/gazebo-models.md

```diff
@@ -0,0 +1,73 @@
+# Gazebo Models Repository (PX4-gazebo-models)
+
+The [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository is used to store all [Gazebo](../sim_gazebo_gz/README.md) models and worlds that are supported by PX4.
+
+- Models are stored in the `/models` directory and worlds are stored in the `/worlds` directory.
+- The [simulation-gazebo](https://github.com/PX4/PX4-gazebo-models/blob/main/simulation-gazebo) Python script is used for starting Gazebo in standalone mode.
+  This script can communicate with a PX4 SITL instance on the same host by default.
+  If the script parameters are set correctly, it can also communicate with any PX4 instance on any machine within the same network.
+
+## Simulation Start-up
+
+The `simulation-gazebo` script does not require any additional libraries and should work out of the box.
+The default `simulation-script` can be run with:
+
+```sh
+python simulation-gazebo
+```
+
+With these default parameters, there will first be a check whether a folder called `/.simulation-gazebo` already exists.
+If if does not, all models and worlds will be downloaded from [PX4 gazebo models repository](https://github.com/PX4/PX4-gazebo-models) and placed there.
+Then, a _gz-server_ instance will be launched using the default grey plane world.
+There are then multiple ways to connect a PX4-enabled vehicle to this instance of _gz-server_.
+
+1. In a new terminal, run PX4 using `make px4_sitl gz_<vehicle>` and you will observe a vehicle appearing in Gazebo.
+2. Use the gazebo resource spawner to look for px4 vehicles on fuel.
+   Drag and drop the vehicle of your choice into Gazebo and then launch PX4 with:
+
+   ```sh
+   PX4_SYS_AUTOSTART=<airframe-number-of-choice> PX4_GZ_MODEL_NAME=<vehicle-of-choice> ./build/px4_sitl_default/bin/px4`
+   ```
+
+   This will connect PX4 to the running instance of Gazebo.
```

Comment:
These options are either/or, not ordered? If so, make them bullets.

Reply:
Ok